spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -11,6 +11,192 @@ We then build an introductory Spring Cloud Data Flow application, discussing som
 
 
 
+[[getting-started-deploying-spring-cloud-dataflow-quick-start]]
+== Quick Start using Docker Compose
+
+As of Spring Cloud Data Flow 1.4, a Docker Compose file is provided to quickly bring up Spring Cloud Data Flow and its dependencies without having to obtain them manually. Docker Compose is required and it's recommended to use the link:https://docs.docker.com/compose/install/[latest version].
+
+. Download the Spring Cloud Data Flow Local Server Docker Compose file:
++
+[source,bash,subs=attributes]
+----
+wget https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/master/spring-cloud-dataflow-server-local/docker-compose.yml
+----
++
+. Start Docker Compose
++
+In the directory where you downloaded `docker-compose.yml`, start the system, as follows:
++
+[source,bash,subs=attributes]
+----
+$ docker-compose up
```

Comment:
Do we need to a put a `NOTE` beneath this command regarding the local Docker registry and the caching fun? And what to do to pull the latest image?

Reply:
yes good catch, i called out `latest` specifically as that should be the only one that changes